src/pf.c

```diff
@@ -35,6 +35,9 @@
 #endif
 
 #include <net/pfvar.h>
+#ifdef __FreeBSD__
```

Comment:
We cannot unconditionally include `<libpfctl.h>` on FreeBSD, since the header is only available when the "libpfctl" package is installed. Please use `HAVE_LIBPFCTL_H` instead.

Reply:
OK, let me try that.